sdk/purview/Management.Purview/Generated/AccountsOperations.cs

```diff
@@ -0,0 +1,2057 @@
+// <auto-generated>
```

Comment:
In order for the ci jobs to function, you need to follow the directory conventions: the directory should be Azure.ResourceManager.Purview, and that shoud also be the project and assembly name

Reply:
https://github.com/Azure/azure-rest-api-specs/pull/12857 thanks for the pointer